### DISCUSSION TITLE
The Invisible Ninja Div

### DISCUSSION THREAD

**Discussion by meviru** [meviru/badge-hunting#11] [Galaxy Trivia] [answered]
Your `<div>` is feeling shy today and wants to be completely invisible, even to the keenest eyes. How do you make it vanish without leaving a trace on the page?

**Comment by DarmorGamz** [comment, marked as answer]
Giving the div a ```display: none``` property should ease it's mind.

**Reply by meviru** [reply]
Congratulations!! 👏👏 You've just earned the [Galaxy Brain](https://github.com/DarmorGamz?tab=achievements&achievement=galaxy-brain) badge. Keep it up, and remember to give the repository a star to show your support! 🎉
